### PR TITLE
GDPR FAQ Update

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation task: https://mattermost.atlassian.net/browse/MM-33042

Updated:
- Overview > Certifications and Compliance Overview > GDPR Compliance > FAQ
   - Added new FAQ topic to answer the question "How do you ensure that personal data stays within the European Union?

### CODE REVIEW

File: source/overview/compliance.rst [hunk 1]
```diff
@@ -193,3 +193,11 @@ Based on our interpretation of `article 49 of GDPR <https://eur-lex.europa.eu/le
 Do you have Fed or Department of Defense (DOD) Certification?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 We are in the process of acquiring Authority to Operate (ATO) and Certificate of Networthiness (CON) certifications.
+
+How do you ensure personal data stays within European Union?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When the customer’s installation of Mattermost is self-hosted, Mattermost will not be processing any personal data under the jurisdiction of the data privacy laws governing within the European Union. The Mattermost support team leverages Zendesk customer service software, which hosts Mattermost information within the United States. For more information on Zendesk, please see their `Privacy and Data Protection <https://www.zendesk.com/company/privacy-and-data-protection/#gdpr-sub>`__ page.
```
Comment (wiersgallak) [inline source/overview/compliance.rst:199]:
```suggestion
When the customer’s installation of Mattermost is self-hosted, Mattermost does not process any personal data under the jurisdiction of the data privacy laws governing within the European Union. The Mattermost support team leverages Zendesk customer service software, which hosts Mattermost information within the United States. For more information on Zendesk, please see their `Privacy and Data Protection <https://www.zendesk.com/company/privacy-and-data-protection/#gdpr-sub>`__ page.
```

**Review by wiersgallak — CHANGES_REQUESTED** [review]

**Review by adamjclarkson — APPROVED** [review]

**Comment by cwarnermm** [comment]
@wiersgallak - Are there additional reviews we'd like this change to go through prior to publication?

**Review by wiersgallak — COMMENTED** [review]
This information came from Jamie if I recall, so I think we are good from that perspective. Wondering if we should refer them back to the Privacy Policy as well in the statement.

**Comment by cwarnermm** [comment]
Thanks, @wiersgallak! I've added a link to our Privacy Policy to the new FAQ topic.

**Review by wiersgallak — APPROVED** [review]
Thanks!

File: source/overview/compliance.rst [hunk 2]
```diff
@@ -193,3 +193,11 @@ Based on our interpretation of `article 49 of GDPR <https://eur-lex.europa.eu/le
...
+
+Zendesk privacy and data protection safeguards notwithstanding, the provision of support services is part of the contractual obligations between Mattermost and its customers. In order for Mattermost to provide such support, a customer must be able to identify as a licensed user, therefore requiring the user to provide personal data to the support agent. Regardless of where the support agent is located, the personal data will indeed be hosted outside of the EU. 
+
+However, pursuant to Section (b) of Article 49 of GDPR, transfers of personal data which are "necessary for the performance of a contract between the data subject and the controller" may be transferred to a third country or international organization. Accordingly these transfers would be done in alignment with the requirements of GDPR. For more information, see our `Mattermost Privacy Policy <https://mattermost.com/privacy-policy/>`__ page.
```
Comment (justinegeffen) [inline source/overview/compliance.rst:203]:
```suggestion
However, pursuant to Section (b) of Article 49 of GDPR, transfers of personal data which are "necessary for the performance of a contract between the data subject and the controller" may be transferred to a third country or international organization. Accordingly these transfers would be done in alignment with the requirements of GDPR. For more information, see our `Mattermost Privacy Policy <https://mattermost.com/privacy-policy/>`__ page.

```

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**